common/src/api/internal/shared.rs

```diff
@@ -82,6 +90,41 @@ pub struct RackNetworkConfig {
     pub bgp: Vec<BgpConfig>,
 }
 
+/// Deprecated, use `RackNetworkConfig` instead. Cannot actually deprecate due to
+/// <https://github.com/serde-rs/serde/issues/2195>
+///
+/// Our first version of `RackNetworkConfig`. If this exists in the bootstore, we
+/// upgrade out of it into `RackNetworkConfigV1` or later versions if possible.
+/// Initial network configuration
+///
+// TODO(AJS): Should this actually exist in the OpenAPI spec or should we just
+// hide the old version inside sled-agent/src/bootstrap/early_networking.rs
```

Comment:
I think since we're not yet doing multiple-API-versions-supported-simultaneously, I would vote for "hide it from OpenAPI" so that the spec only shows what should currently be provided / returned.

Reply:
Will do!